RodrigoCostaTA/university-management-system
# MongoDB setup

## Title:
Feature: Added Mongoose for MongoDB Integration

## Description:

### Context:
This PR aims to add Mongoose for MongoDB integration.

### Changes Made:
1. Added Mongoose as a dependency (`npm install mongoose`).
2. Removed TypeORM-related files: `ormconfig.json`, `entities/*`.
3. Updated MongoDB connection code in `index.ts` to use Mongoose.
4. Updated `Student.ts` to define the Mongoose schema for the Student model.

### How to Test:
1. Ensure MongoDB is running locally or update the connection URL in `index.ts` accordingly.
2. Run the project using `node dist/index.js`.
3. Verify that CRUD operations for the Student model work as expected.

## Checklist:
- [ ] All checks and tests pass.
- [ ] Dependencies on other PRs are resolved.
